### PR TITLE
Add option to show traffic to maps control

### PR DESCRIPTION
**Pull request by krdmllr**
### Description of Change ###

Add the property `TrafficEnabled` to the Maps control.

### API Changes ### 
Added:
```
public static readonly BindableProperty TrafficEnabledProperty;

public bool TrafficEnabled{ get; set; }
```

### Platforms Affected ### 

-iOS
-macOS
-Android
-UWP
-Tizen (not tested!)

not supported by native control:
-WPF (workaround using XAML islands would be possible but only available on Win10)
-GTK

### Behavioral/Visual Changes ###

None, disabled by default.

### Before/After Screenshots ### 

- UWP
image
- iOS
grafik
- macOS
grafik

- Android
image

- Tizen

### Testing Procedure ###

Testable in the control gallery -> Maps Control (Legacy)

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Maps/Map.cs [hunk 1]
```diff
@@ -15,6 +15,8 @@ public class Map : View, IEnumerable<Pin>
 
 		public static readonly BindableProperty IsShowingUserProperty = BindableProperty.Create("IsShowingUser", typeof(bool), typeof(Map), default(bool));
 
+		public static readonly BindableProperty IsShowingTrafficProperty = BindableProperty.Create("IsShowingTraffic", typeof(bool), typeof(Map), default(bool));
```
[the remark this answers is not in the document]
Comment (krdmllr) [reply Xamarin.Forms.Maps/Map.cs]: I would agree with both, Tizen and Android use `TrafficEnabled` which you could consider aswell.

**Comment by krdmllr** [comment]
Added samples for ios/macos. I cant test it on tizen.

File: Xamarin.Forms.Maps/Map.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (samhouts) [reply Xamarin.Forms.Maps/Map.cs]: I like `TrafficEnabled`. Sort of goes with the other properties but omits the unnecessary extra verb. :)

[the remark this answers is not in the document]
Comment (krdmllr) [reply Xamarin.Forms.Maps/Map.cs]: Should we use `TrafficEnabled` or will you discuss this further internally?

**Comment by rmarinho** [comment]
@krdmllr do you mind rename to `TrafficEnabled` ?

**Comment by krdmllr** [comment]
@rmarinho changed it to TrafficEnabled and rebased on master.

**Comment by krdmllr** [comment]
I dont think the test issues are related to the PR. Can you look into this?

**Comment by rmarinho** [comment]
@krdmllr  can you rebase again to see if we can make sure tests pass? thanks

**Comment by krdmllr** [comment]
@rmarinho done

**Pull request merged**